.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+# Please see the documentation for all configuration options:
+# https://docs.github.com/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+  - package-ecosystem: "mix"
```

Comment:
I don't think this is necessary. 

Updating packages other than dev dependencies is just noise and doesn't help the consumer if the library (mix.lock is not used by consumers)